aten/src/ATen/core/ivalue.h

```diff
@@ -1179,7 +1179,7 @@ struct TORCH_API IValue final {
       case Tag::Device:
         return false;
       case Tag::Stream:
-        return false;
+        return true;
```

Comment:
huh? Surprised by this change... Can't believe this didn't cause issues before...